mpt/impl/imap-mailbox/core/src/main/java/org/apache/james/mpt/host/JamesImapHostSystem.java

```diff
@@ -57,6 +76,12 @@ public void configure(ImapDecoder decoder, ImapEncoder encoder,
         this.processor = processor;
     }
 
+    @Override
+    public boolean addUser(String user, String password) throws Exception {
+        memoryUsersRepository.addUser(user, password);
+        return true;
```

Comment:
can happen in ExternalHostSystem. I agree that it could be handled with exceptions, but that's not in the scope of this PR.